tools/cockpit.spec

```diff
@@ -107,10 +111,13 @@ Patch102:       0002-selinux-temporary-remove-setroubleshoot-section.patch
 %endif
 
 # Ship custom SELinux policy (but not for cockpit-appstream)
-%if 0%{?rhel} >= 9 || 0%{?fedora} || 0%{?suse_version}
+# SLES / Leap 15 does not have selinux-policy. TW and SLE Micro does.
+%define with_selinux 0
```

Comment:
These 2 lines are not needed. Everywhere the guards are

`%if 0%{?with_selinux}`

which means by default it will build without selinux. Defining it to 0 is not necessary here.

Reply:
removed